src/gobapi/storage.py

```diff
@@ -228,8 +269,28 @@ def _get_convert_for_table(table, filter=None):
     :return:
     """
     def convert(entity):
-        # Use the sqltypes to get the correct gobtype and return a dict
-        hal_entity = {column.name: _to_gob_value(entity, column.name, type(column.type)) for column in columns}
+        def resolve_column(column):
+            """Resolves the name and value of the given column for entity using _to_gob_value
+            Uses attributes set by _add_resolve_attrs_to_columns to determine how values are resolved.
+
+            If 'attribute' is set on column, pass 'attribute' to _to_gob_value, in which case the GOB type will be
+            used. Otherwise, pass SQL type of the column.
+
+            Passes 'authority' to _to_gob_value to be able to resolve secured values.
+            """
+            value = _to_gob_value(
+                entity,
+                column.name,
+                getattr(column, 'attribute') if hasattr(column, 'attribute') else type(column.type),
+                authority=getattr(column, 'authority', None),
```

Comment:
You can use a default value: getattr(column, 'attribute', type(column.type))

Reply:
Done :)